main.go

```diff
@@ -143,6 +143,7 @@ func mainWithError() error {
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.CrtFile, "", "Certificate file path to use to authenticate with Kubernetes.")
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.KeyFile, "", "Key file path to use to authenticate with Kubernetes.")
 
+	daemonCommand.PersistentFlags().String(f.Service.Resource.CertificateDirectory, "/certs", "Directory in which to store certificates.")
```

Comment:
From a client side I would prefer to do `f.Service.Resource.Certificate.Directory`.

Reply:
This feels like too many blocks to me.